src/Evolution/Systems/GrMhd/ValenciaDivClean/Subcell/TimeDerivative.hpp

```diff
@@ -315,18 +319,35 @@ struct TimeDerivative {
             // NormalCovectorAndMagnitude tag in the DataBox right now to avoid
             // conflicts with the DG solver. We can explore in the future if
             // it's possible to reuse that allocation.
-            const Scalar<DataVector> normalization{
-                sqrt(get<gr::Tags::InverseSpatialMetric<DataVector, 3>>(
-                         vars_upper_face)
-                         .get(i, i))};
-
+            //
+            // The unnormalized normal vector is
+            // n_j = d \xi^{\hat i}/dx^j
+            // with "i" the current face.
             tnsr::i<DataVector, 3, Frame::Inertial> lower_outward_conormal{
                 reconstructed_num_pts, 0.0};
-            lower_outward_conormal.get(i) = 1.0 / get(normalization);
+            for (size_t j = 0; j < 3; j++) {
+              lower_outward_conormal.get(j) =
+                  evolution::dg::subcell::fd::project_to_face(
```

Comment:
rename project_to_faces (as I was confused about how what the target of the projection is)